### DISCUSSION TITLE
how does IIS (inter-server-sync) work

### DISCUSSION THREAD

**Discussion by fockarty** [uyuni-project/uyuni#9056] [Q&A] [answered]
I am trying to get inter-server-sync op and running but is there some more information about what it does and howto configure it? in the uyuni docs there is not much info about it. I can sync with the master but no date show up at the slave, same for systems. If i join a server at the master it doesn't show up at the slaves.. Am i doing something wrong or do i have the wrong expectations?

**Comment by rjmateus** [comment]
you have two versions on ISS. Version 1 is the old one and it run an online sync, but you need to manually configure it [1]. 
ISS v2 works as an online export/import. Would would need to export on the source server, copy the export folder to the target server, and run import [2].
[1] https://www.uyuni-project.org/uyuni-docs/en/uyuni/administration/iss.html
[2] https://www.uyuni-project.org/uyuni-docs/en/uyuni/administration/iss_v2.html

**Comment by fockarty** [comment]
yes, i have read and followed both, but data doesn't get replicated. As far as 1 understood it doesn't replicate after the initial replication. I would expect iss could function as a failover construction, is that possible?

**Reply by rjmateus** [reply]
I'm not sure about what you mean for a failover construction, could you elaborate on that?

**Reply by fockarty** [reply]
him with failover i mean that if the first uyuni server fails, the second can take over. since uyuni also functions as repo that might be handy. I just read a document from redhat which says that iss is intended for air-gaped installations not failover. can you confirm that?

**Reply by rjmateus** [reply, marked as answer]
yes I can confirm that. We don't have an official mechanism for failover scenarios. However, with the containerization version you can just copy the podman volumes folder to a different location (or have an extra disk mounted in the server in the podman volumes location).
If you want to recover to a different machine, you need to deploy the backup of that folder, or mount the data disk in there, and re-install the server. It will re-use the existing data, and then you just need to change DNS and IP to point to the new server and you can keep all the data and systems information.
